maint-lib/stagelib/envglobals.py

```diff
@@ -48,7 +48,18 @@ def _verifyRequiredEnvVar(varname):
         sys.stderr.write(_NOT_SET_TEXT.format(varname))
         sys.stderr.write(_REQUIRED_VAR_TEXT.format(varname))
         sys.exit(1)
+    # We need to extract a possible CEPH_POINT_RELEASE from CEPH_VERSION
+    # It could be under "package_name-package_version" or "package_name=package_version"
+    if varname == "CEPH_VERSION":
+        os.environ["CEPH_POINT_RELEASE"] = " "
```

Comment:
This function is a verify function and should not modify variables. I am opposed to doing this in Python, but if you must, follow the formula I have followed before with `exportGitInfoEnvVars`.

Also, do not forget to add a test for this new functionality into `tests/stage-test`

Reply:
I will

Reply:
Good catch.
Updated by creating populateCephPointRelease() which take care of it and so verifyRequiredEnvVars is untouched and the printing done in the proper order.